frontend/app/src/components/defi/Lending.vue

```diff
@@ -11,7 +11,16 @@
           :loading="allLoading"
           title="Lending"
           @refresh="refresh()"
-        />
+        >
+          <confirmable-reset
+            :loading="allLoading"
+            tooltip="Refreshes the data ignoring cached Aave historical entries"
+            @reset="reset()"
+          >
+            This action will ignore any cached entries in Aave history and will
```

Comment:
I would rewrite:
`This action will ignore any cached entries in Aave history and will fetch everything again. Do you want to continue?`
to
`This action will overwrite any Aave history cached entries in the DB and will fetch everything again. Depending on the number of accounts it may take a long time. Are you sure you want to continue?`